tests/OnlineSales.Tests/BaseTest.cs

```diff
@@ -109,22 +130,35 @@ protected async Task<HttpResponseMessage> GetTest(string url, HttpStatusCode exp
             return null;
         }
     }
+
+    protected async Task<Stream?> GetImageTest(string url, HttpStatusCode expectedCode = HttpStatusCode.OK, string authToken = "Success")

```

Comment:
This method should be moved away from BaseClass. Please put it to ImagesTests.cs as it it very specific to that particular test class. We do not need it in any other test cases.